client/components/time-since/index.jsx

```diff
@@ -1,40 +1,36 @@
 import moment from 'moment';
-import { PureComponent } from 'react';
-import humanDate from 'calypso/lib/human-date';
-import { Interval, EVERY_TEN_SECONDS } from 'calypso/lib/interval';
-import smartSetState from 'calypso/lib/react-smart-set-state';
+import { useState, useEffect } from 'react';
+import getHumanDate from 'calypso/lib/human-date';
+import { useInterval, EVERY_TEN_SECONDS } from 'calypso/lib/interval';
 
-export default class TimeSince extends PureComponent {
-	smartSetState = smartSetState;
+function TimeSince( { className, date, dateFormat } ) {
+	const [ humanDate, setHumanDate ] = useState( '' );
+	const [ fullDate, setFullDate ] = useState( '' );
 
-	UNSAFE_componentWillMount() {
-		this.update();
-	}
+	function update( newDate ) {
```

Comment:
`update` shouldn't need the `newDate` param. It's always the `props.date` value.

The original class component was different because one lifecycle method used `this.props.date`, while the other used `nextProps.date`.